### PR TITLE
feat: add internal analytics about new method & profiles

### PR DESCRIPTION
**Pull request by jakub-roch**
Add internal analytics about usage of new method & profiles feature

### CODE REVIEW

File: src/video-player.js [hunk 1]
```diff
@@ -55,6 +55,7 @@ class VideoPlayer extends Utils.mixin(Eventable) {
     this.videoElement = getResolveVideoElement(elem);
 
     this.options = extractOptions(this.videoElement, initOptions);
+    console.log('1111', this.options);
```
Comment (ehab-cl) [inline src/video-player.js:58]:
```suggestion
```

File: src/video-player.js [hunk 2]
```diff
@@ -112,6 +113,7 @@ class VideoPlayer extends Utils.mixin(Eventable) {
       return;
     }
     try {
+      const internalAnalyticsMetadata = this.options.videojsOptions._internalAnalyticsMetadata || {};
```
Comment (ehab-cl) [inline src/video-player.js:116]:
```suggestion
      const internalAnalyticsMetadata = this.options.videojsOptions._internalAnalyticsMetadata ?? {};
```

Comment (ehab-cl) [reply src/video-player.js:116]: does not also make sense that something that is Cloudinary related is stored under `videojsOptions` ?

**Review by ehab-cl — APPROVED** [review]

File: src/video-player.js (hunk 1 above)
Comment (jakub-roch) [reply src/video-player.js]: 👍

File: src/video-player.js (hunk 2 above)
Comment (jakub-roch) [reply src/video-player.js]: good point, I moved it to playerOptions as its internal property not from videojs

**Pull request merged**